core/types/block_test.go

```diff
@@ -0,0 +1,370 @@
+package types
+
+import (
+	"bytes"
+	"encoding/binary"
+	"testing"
+	"time"
+
+	"github.com/kwilteam/kwil-db/core/crypto"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetRawBlockTx(t *testing.T) {
```

Comment:
I actually did a git mv, but added enough new tests that git decided it wasn't similar enough to link.  :shrug: